jupyter-docker/run-jupyter-local.sh

```diff
@@ -12,8 +12,9 @@ start () {
     fi
 
     echo "Starting Jupyter server container..."
-    docker create -it --name ${CONTAINER} --rm -p 8001:8001 $DOCKER_IMG
+    docker create -it --name ${CONTAINER} --rm -p 8001:8001 $DOCKER_IMG 'export PYTHONPATH=$PYTHONPATH:/etc/jupyter/custom;start-notebook.sh'
```

Comment:
This is a total guess, based on the documentation [here](https://github.com/jupyter/docker-stacks/tree/master/minimal-notebook)

Reply:
I'll try it out at some point. There might be other problems with this script as well; I remember having to hack at it a bit to get it working.